src/components/core/DpDashboardTaskCard.vue

```diff
@@ -53,7 +68,7 @@ export default {
 
   computed: {
     userFilteredSegmentUrl () {
-      return Routing.generate('dplan_segments_list', { procedureId: this.procedureId }) + '/' + this.userHash
+      return Routing.generate(this.segmentsListRoute, { procedureId: this.procedureId }) + '/' + this.userHash
```

Comment:
I think we want to pass the complete route - the result of Routing.generate - not just the String that has to be generated.

Reply:
See:
https://github.com/demos-europe/demosplan-ui/pull/91#discussion_r1093328892
https://github.com/demos-europe/demosplan-ui/pull/91#discussion_r1093359398